docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,43 @@
+| cumulative           | `Cumulative voting`_, where each voter can distribute their vote to up to        |
+|                      | *N* candidates.                                                                  |
 +----------------------+----------------------------------------------------------------------------------+
-| n-of-m               |                                                                                  |
+| majority             | A 1-of-m method where the winner needs more than 50% of the vote to be elected.  |
 +----------------------+----------------------------------------------------------------------------------+
-| plurality            |                                                                                  |
+| n-of-m               | A method where each voter can select up to *N* candidates.                       |
 +----------------------+----------------------------------------------------------------------------------+
-| proportional         |                                                                                  |
+| plurality            | A 1-of-m method where the candidate with the most votes is elected, regardless   |
+|                      | of whether the candidate has more than 50% of the vote.                          |
 +----------------------+----------------------------------------------------------------------------------+
-| range                |                                                                                  |
+| range                | `Range voting`_, where each voter can select a score for each candidate.         |
 +----------------------+----------------------------------------------------------------------------------+
-| rcv                  |                                                                                  |
+| rcv                  | `Ranked choice voting`_ (RCV), where each voter can rank the candidates, and     |
+|                      | the ballots are counted in rounds.  Also known as instant-runoff voting (IRV)    |
+|                      | and the single transferable vote (STV).                                          |
 +----------------------+----------------------------------------------------------------------------------+
-| rcv-borda            |                                                                                  |
+| rcv-borda            | `Borda count`_, where each voter can rank the candidates, and the rankings are   |
+|                      | assigned point values.                                                           |
 +----------------------+----------------------------------------------------------------------------------+
-| super-majority       |                                                                                  |
+| super-majority       | A 1-of-m method where the winner needs more than some predetermined fraction     |
```

Comment:
Same as my previous two replies [here](https://github.com/votinginfoproject/vip-specification/pull/184/files#r31302240) and [here](https://github.com/votinginfoproject/vip-specification/pull/184/files#r31303001) in the "plurality" and "majority" cases.  I think we should resolve the ambiguity now rather than leave everyone else scratching their heads as to which enum value to choose when more than one is applicable.